### ISSUE TITLE
GFM Strikethrough not always working 

### ISSUE THREAD

**Issue by arjan** [RobertDober/earmark_parser#115]
Library version: 1.4.28

```
EarmarkParser.as_ast("~~x~~")
```
results in expected:
```
{:ok, [{"p", [], [{"del", [], ["x"], %{}}], %{}}], []}
```
However, when the strikethrough is part of a bigger string it fails:

```
EarmarkParser.as_ast("a ~~x~~")
```

The strikethrough output remains unparsed:

```
{:ok, [{"p", [], ["a ~~x~~"], %{}}], []}
```

**Comment by RobertDober** [comment]
Ty for reporting this, I'll have a look ASAP.

**Comment by RobertDober** [comment]
Just released fix in 1.4.29
